DigitalRain/Streams/RaindropStream.cs

```diff
@@ -5,23 +5,22 @@
 namespace DigitalRain.Raindrops
 {
     using Columns;
+    using GameUtilities;
 
-    public class RaindropStream
+    public class RaindropStream: IGameObject
     {
-        private StandardRaindropFactory _raindropFactory;
-        private float _width;
-        private float _fontHeight;
+        private readonly StandardRaindropFactory _raindropFactory;
+        private readonly float _fontHeight;
         private double? _startTimeInSeconds;
-        private float _speedInPixelsPerSecond;
+        private readonly float _speedInPixelsPerSecond;
         private double _unboundDistanceFallenInPixels;
         private int _raindropCount;
-        List<StandardRaindrop> _raindrops;
+        readonly List<StandardRaindrop> _raindrops;
 
-        public RaindropStream(StandardRaindropFactory raindropFactory, Column column, float width, float speedInPixelsPerSecond, float fontHeight)
+        public RaindropStream(StandardRaindropFactory raindropFactory, Column column, float speedInPixelsPerSecond, float fontHeight)
```

Comment:
So here's why I thought we would need width.  I was going to make the raindrops draw themselves to completely fill the column space they get given, by default.  So width would be how you tell them not to do that. This would allow us to produce a precise spacing between the streams.

Reply:
We can change this in the future if we need it. It was unused so I just yeet'ed it